### PR TITLE
Funeral Of The Dead Butterflies

### PR DESCRIPTION
**Pull request by dsmith328**
## About The Pull Request
Adds Funeral of the Dead Butterflies. Code is by ReaperOni. He was just struggling with squashing a PR so I took the code and double checked stuff. Leave any issues here and I'll make sure to fix them.

## Why It's Good For The Game
New HE is good, cool pistols too.

## Changelog
:cl:
add: Funeral, his EGO, his EGO Gift, and his Records
/:cl:

### CODE REVIEW

File: code/game/objects/effects/temporary_visuals/miscellaneous.dm [hunk 1]
```diff
@@ -676,6 +676,15 @@
 /obj/effect/temp_visual/flesh/proc/fade_out()
 	animate(src, alpha = 0, time = (duration - 4 SECONDS))
 
+/obj/effect/temp_visual/funeral_swarm
+	name = "funeral swarm"
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "funeral_swarm"
+	layer = BELOW_MOB_LAYER
+	pixel_x = -8
```
Comment (EgorDinamit) [inline code/game/objects/effects/temporary_visuals/miscellaneous.dm:684]: Why did you shift its position if it's a 32x32 image?

Comment (dsmith328) [reply code/game/objects/effects/temporary_visuals/miscellaneous.dm:684]: Didn't catch that

**Comment by Kitsunemitsu** [comment]
This has massive range and the most HP out of all HEs.
Please bring the HP down to 1000-1100

**Comment by Kitsunemitsu** [comment]
Bullet is too fast, you can't really see it.
It should be as fast as bullets are now.

All bullets will be getting a speed buff soon.

**Comment by ReapedOni** [comment]
> This has massive range and the most HP out of all HEs. Please bring the HP down to 1000-1100
The HP is to scale with the rest of the abnormalities, and he's a threatening HE regardless if handled incorrectly. When handled correctly, just like in LC he's a fucking pushover.

**Comment by ReapedOni** [comment]
> Bullet is too fast, you can't really see it. It should be as fast as bullets are now.
> 
> All bullets will be getting a speed buff soon.

this is true, but since i'm not a contributor for the pr i can't quite edit the change in.

**Comment by dsmith328** [comment]
This should just need the Directional Sprite for Coffin

**Comment by dsmith328** [comment]
Directional Coffin sprites added, Thanks InsightfulParasite!

**Comment by EgorDinamit** [comment]
Fix conflicts.

**Comment by dsmith328** [comment]
> Fix conflicts.

After EGO Gifts is merged. I knew these would arise.

**Pull request merged**